en/sensor/uart_ulanding_radar.md

```diff
@@ -20,8 +20,9 @@ You can simply add the following line to an [extras.txt](../advanced/system_star
 ulanding_radar start /dev/serial_port
 ```
 
-In the upper command you will have to replace the last argument with the serial port you have connected the hardware to.
-If you don't specify any port the driver will use /dev/ttyS2 which is the TELEM2 port on Pixhawk.
+In the above command you will have to replace the last argument with the serial port you have connected the hardware to.
+If you don't specify any port the driver will use /dev/ttyS2 which is the TELEM2 port on Pixhawk. If either of the OcPoc config
+files are used (posix_ocpoc_cross.cmake or posix_ocpoc_ubuntu.cmake) the default port will be /dev/ttyS6.
 
 **Warning**
```

Comment:
Best way to do warnings is to use the syntax like below (this wraps the text in a very visible box).

```
> **Warning** This is the text in the warning ...
```